Author: Holger Jaekel <holger.jaekel@gmx.de>
Summary: ignore depreciation warnings for ares_fds and ares_process
----

--- a/src/net/dns.cc
+++ b/src/net/dns.cc
@@ -501,7 +501,10 @@
             FD_ZERO(&readers);
             FD_ZERO(&writers);
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
             n = ares_fds(_channel, &readers, &writers);
+#pragma GCC diagnostic pop
 
             dns_log.trace("ares_fds: {}", n);
 
@@ -534,7 +537,10 @@
                 }
             }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
             ares_process(_channel, &readers, &writers);
+#pragma GCC diagnostic pop
         } while (n != 0);
     }
 
